bundles/org.openhab.binding.sonnen/README.md

```diff
@@ -12,6 +14,7 @@ More information about the sonnen battery can be found [here](https://sonnen.de/
 ## Thing Configuration
 
 Only the parameter `hostIP` is required; this is the IP address of the sonnen battery in your local network.
+If you want to use the V2 API of the vendor sonnen which supports more channels you need to provide the `Authentication Token`.
```

Comment:
Actual parameter name:
```suggestion
If you want to use the V2 API of the vendor sonnen which supports more channels you need to provide the `authToken`.
```

It might also be an idea to reformat this into a nice table describing both parameters.

Reply:
ok